src/hooks/tests/usePatchStatusSelect.test.ts

```diff
@@ -259,6 +297,15 @@ const patchBuildVariants = [
   },
 ];
 
+const versionId = "mainVersion";
+const childVersion = [
```

Comment:
Could we include some tests that show how this works with childVersions. If this hook supports them.

Reply:
would it be okay to do that as part of https://jira.mongodb.org/browse/EVG-15414?